readthedocs/oauth/services/bitbucket.py

```diff
@@ -147,6 +148,8 @@ def create_repository(self, fields, privacy=None, organization=None):
 
             repo.organization = organization
             repo.name = fields['name']
+            repo.remote_id = fields['uuid']
```

Comment:
Bitbucket returns a weird id (includes curly braces). Don't know if it should be stripped or be this way.

``"uuid": "{39c6eaba-9b71-47f1-a88c-249a656a60f8}"``

Reply:
Weird... I wouldn't touch it and rely on their weirdness thing. I hope it's always returned in the same way :)